lib/sqlalchemy/sql/elements.py

```diff
@@ -80,12 +79,20 @@
 from ..util.typing import Self
 
 if typing.TYPE_CHECKING:
+    from re import Match
+
+    from mypy_extensions import NoReturn
+
+    from ._py_util import cache_anon_map
```

Comment:
**mike bayer** (zzzeek) wrote:

import this from visitors

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4512